src/templates/community-participate.ejs

```diff
@@ -0,0 +1,100 @@
+              <a href="https://webchat.freenode.net/?channels=%23pyramid&uio=d4">#pyramid</a>
+              IRC channel on <a href="http://freenode.net/">FreeNode</a>.
+            </li>
+          </ul>
+          <span id="make-sure-your-bug-hasn-t-already-been-reported" />
+          <h3>3) Make sure your bug hasn't already been reported</h3>
+          <p>Before creating an issue, search through the appropriate Issue tracker on
+          <a href="http://github.com/">GitHub</a>. If you find a bug like yours, and it hasn't
+          yet been fixed, check to see if you have new information that could be reported to
+          help the developers fix it.</p>
+          <span id="collect-information-about-the-bug" />
+          <h3>4) Collect information about the bug</h3>
+          <p>To provide the developers with enough information to fix a bug, they'll need
+            to be able to easily repeat the conditions and observe it themselves.  The
+            the likelihood that a bug will be fixed is directly correlated to the
+            developer's ability to create the conditions under which the bug may be
+            observed.</p>
+          <p>Often a bug report should include a Python traceback (<a href="http://pastebin.com/TyaPKpt9"
+            >see a Python traceback</a>).  We will also need to know what platform you're running
+            (Windows, OSX, Linux), and which Python interpreter you're running if its not CPython
+            (e.g. Jython, Google App Engine).</p>
+          <span id="submit-the-bug" />
+          <h3>5) Submit the bug</h3>
+          <p>By default <a href="http://github.com/">GitHub</a> will email you to let you know
+          when new comments have been made on your bug report. In the event you've turned this
+          feature off, you should check back on occasion to ensure you don't miss any questions
+          a developer trying to fix the bug might ask.</p>
+        </div>
+
+        <div class="col-sm-4">
+          <span id="issue-trackers" />
+          <h2 class="title text-blue">Issue Trackers</h2>
+          <p>Bugs are reported and tracked via <a href="http://github.com/">GitHub</a>'s issue
+            trackers. Each Pylons Project has its own issue tracker:</p>
+          <ul>
+            <li><a href="https://github.com/Pylons/pyramid/issues">pyramid issue tracker</a></li>
+            <li><a href="https://github.com/Pylons/pyramid_beaker/issues">pyramid_beaker issue tracker</a></li>
+            <li><a href="https://github.com/Pylons/pyramid_xmlrpc/issues">pyramid_xmlrpc issue tracker</a></li>
+            <li><a href="https://github.com/Pylons/pyramid_jinja2/issues">pyramid_jinja2 issue tracker</a></li>
+            <li><a href="https://github.com/Pylons/pylonshq/issues">Pylons Project issue tracker</a> (All
+              bugs with the pylonshq.com/pylonsproject.org website should be reported here.)</li>
```

Comment:
The above two should use the new repo/URL:
* https://github.com/Pylons/pylonsproject.org/issues
* bugs with the www.pylonsproject.org website